src/main/js/App.js

```diff
@@ -300,8 +337,24 @@ const App = () => {
                              variant="outlined"
                              label="Survey Id"
                              size="small"
-                             onChange={e => setUserSurveyName(e.target.value)}
+                             onChange={e => setUserSurveyId(e.target.value)}
                     />
+                     <FormControl className="qq-app mv qq-app__seletIds_select">
```

Comment:
Typo in "seletIds". Also, for consistency we should use dashes to separate words instead of capital letters.